modules/appeals_api/config/routes.rb

```diff
@@ -175,7 +175,7 @@
       get :docs, to: '/appeals_api/docs/v2/docs#sc'
 
       namespace :forms do
-        resources '200995', only: %i[create show], controller: controller_path do
+        resources '200995', only: %i[create index show], controller: controller_path do
```

Comment:
@kristen-brown A bare `index` (without an ICN supplied) is not supported at this time - though there are probably use cases for it, we just haven't gone through any of them w/ product. The `index` with an ICN supplied is meant for a veteran to see the submission status of any decision reviews they've submitted through our API. 

Since I'm thinking about it, I don't recall if our index action filters the results to the appellant, and it should. Veterans should not be able to see decision reviews submitted by a non-veteran claimant. After PII is removed I don't think we have a way to determine whether a decision review was submitted by a non-veteran claimant...